tox.ini

```diff
@@ -1,22 +1,27 @@
 [tox]
-envlist = 
-    py26,py27,py32,py33,pypy,cover
+envlist =
+    py27,py36,py37,py38,py39,pypy27,pypy3,coverage
 
 [testenv]
-commands = 
-    python setup.py dev
-    python setup.py test
-deps =
+extras =
+    tests
+setenv =
+    COVERAGE_FILE=.coverage.{envname}
+commands =
+    python --version
+    coverage run -m pytest {posargs:}
```

Comment:
Instead of running coverage directly you can also set flags for pytest to do so by default.

Reply:
I think by default `pytest` doesn't support running coverage without adding 3rd party plugins (`pytest-cov`?). I think this works pretty well without introducing any 3rd parties that aren't strictly necessary to get the job done.

Reply:
Okay. We've installed pytest-cov in most of our other projects, but part of that may have also been with issues in coverage and running on Windows due to python interpreter finding issues with sub-processes.

I would prefer we use `pytest-cov` and handle things the same way in each project, but I won't let this hold up the PR.